src/sentry/testutils/cases.py

```diff
@@ -1710,7 +1710,7 @@ def tag_value(name):
 
         if type == "set":
             # Relay uses a different hashing algorithm, but that's ok
-            value = [int.from_bytes(hashlib.md5(str(value).encode()).digest()[:8], "big")]
+            value = [int.from_bytes(hashlib.md5(str(value).encode()).digest()[:4], "big")]
```

Comment:
Honestly this really should be little endian and not big endian, but I dunno if I want to risk opening another can of worms

Reply:
Does the test still pass if we change it to little endian?